modules/configure-pagerduty.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-pagerduty.adoc
+:_module-type: PROCEDURE
+[id="configure-pagerduty_{context}"]
+= Configuring PagerDuty
+
+[role="_abstract"]
+Start by creating a new service in PagerDuty, and get the integration key.
+
+.Procedure
+. Navigate to *Configuration* -> *Services*.
+. Select *Add Services*.
+. Under *General Settings*, specify a *Name* and *Description*.
+. Under *Integration Setting*, click *Use our API Directly* with *Events v2 API* selected for the *Integration Type* drop-down menu.
+. Under *Incident Settings*, select an *Escalation Policy*, and configure notification settings and incident timeouts.
+. Accept default settings for *Incident Behavior* and *Alert Grouping*, or configure them as required.
+. Click *Add Service*.
+. From the *Service Details* page, note down the *Integration Key*.
```

Comment:
"note down" --> "make note of" or maybe "record" or "copy"?
(When I first read "note down", I was thinking it meant click the down arrow. 😀 )